projects/index.md

```diff
@@ -56,6 +56,7 @@ comments: false
 * [grape_api_signature](https://github.com/faber-lotto/grape_api_signature): A RACK-Based AWS Style signature authentication for grape API.
 * [wine_bouncer](https://github.com/antek-drzewiecki/wine_bouncer): Doorkeeper OAuth2 authorization middleware for grape APIs.
 * [grape-tokeeo](https://github.com/wawandco/grape-tokeeo): A simple token authentication gem for Grape.
+* [grape_token_auth](https://github.com/mcordell/grape_token_auth): Grape token auth gem for frontend JS apps (ng-token-auth compatible)
```

Comment:
Missing a period at the end, maybe change JS to JavaScript, and please link `ng-token-auth`. Thanks.